### PR TITLE
Link latest performance blog on performance benchmarks page

### PR DESCRIPTION
**Pull request by rishabh6788**
### Description
Link latest performance blog on performance benchmarks page. 
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by rishabh6788** [comment]
@kolchfa-aws Can you please review the PR?

File: benchmarks/index.md [hunk 1]
```diff
@@ -61,6 +61,9 @@ benchmark_height_mobile: 6000
 </p>
 <p> You can also view the open GitHub issues tagged to performance label <a href ="https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Aopensearch-project+label%3Aperformance+">here</a>.
 </p>
+<p>
+    Follow the link to read more about <a href="https://opensearch.org/blog/opensearch-performance-2.14/">OpenSearch’s performance improvements since launch.</a>
```
Comment (kolchfa-aws) [inline benchmarks/index.md:65]:
```suggestion
    To learn more about OpenSearch performance improvements through version 2.14, read <a href="https://opensearch.org/blog/opensearch-performance-2.14/">this blog post</a>.
```

**Comment by rishabh6788** [comment]
Sorry for the delay. Add requested changes. @kolchfa-aws

**Review by kolchfa-aws — APPROVED** [review]
LGTM

**Pull request merged**